wire-ios/Wire-iOS/Sources/Components/Settings.swift

```diff
@@ -152,7 +152,7 @@ class Settings {
         NotificationCenter.default.addObserver(self, selector: #selector(applicationDidEnterBackground(_:)), name: UIApplication.didEnterBackgroundNotification, object: nil)
     }
 
-    func migrateAppCenterAndOptOutSettingsToSharedDefaults() {
+    func migrateAppCenterAndOptOutSettingsToSharedDefaults() { // still needed?
         if !defaults.bool(forKey: SettingKey.didMigrateHockeySettingInitially.rawValue) {
             ExtensionSettings.shared.disableLinkPreviews = Settings.disableLinkPreviews
```

Comment:
do we set the disableLinkPreviews else where?

Reply:
`Settings.disableLinkPreviews` reads and writes the value.